packages/client/src/config.ts

```diff
@@ -364,6 +374,8 @@ export class Config {
   public static readonly MINPEERS_DEFAULT = 1
   public static readonly MAXPEERS_DEFAULT = 25
   public static readonly DNSADDR_DEFAULT = '8.8.8.8'
+  public static readonly RPC_ETH_MAXPAYLOAD_DEFAULT = '11mb'
```

Comment:
The previous default was 11mb, have also set this again for the `eth` namespace. `5mb` would be low enough for almost all use cases, but not for blobs (this test failed on the CI). It should now pass :smile: :+1:

Reply:
Ok, it still fails, have to take a deeper look here :thinking: